include/utils.php

```diff
@@ -416,3 +418,16 @@ function tempnamWithCheck($dir, $prefix) {
 
 	return $tmp;
 }
+
+// }}}
+
+// {{{ renderTemplateNoRepo
+
+function renderTemplateNoRepo($view)
```

Comment:
Any reason you didn't put this into `template.php`, which already contains a `renderTemplate`? I suggest adding it after `renderTemplate` there to keep things in context.

Reply:
Yeah i dont remember the thought process. Possibly i thought all small utils to be there. Anyways I moved it to template below the renderTemplate function.